types/react-native/index.d.ts

```diff
@@ -3629,6 +3629,12 @@ interface ImagePropsAndroid {
      * Duration of fade in animation.
      */
     fadeDuration?: number;
+
+    /**
+     * Required sizes if using native image resources on Android
+     */
+    width?: number;
+    height?: number;
```

Comment:
Can you please add doc for both properties separately? Currently the doc will only show up for the `width` property. I also think that “native image resources” is a bit too vague, we want people to read it and immediately understand it applies to their situation, so can you expand it a bit and use Android terminology (drawable)? And perhaps consider linking to the blog post that explains why this is needed.

Reply:
Thanks for the feedback, implemented 👍